server/apiv2/handlers/tso_keyspace_group.go

```diff
@@ -186,10 +189,15 @@ func SplitKeyspaceGroupByID(c *gin.Context) {
 		c.AbortWithStatusJSON(http.StatusBadRequest, "invalid keyspace group id")
 		return
 	}
-	if len(splitParams.Keyspaces) == 0 {
+	if len(splitParams.Keyspaces) == 0 && splitParams.StartKeyspaceID == 0 && splitParams.EndKeyspaceID == 0 {
```

Comment:
If we only give the end key, will it move all keyspace to a new group?

Reply:
Yes, it will move all keyspaces in the source group less than the end keyspace ID to the new group.

Reply:
Do we need to make sure the default keyspace 0 won't be moved?

Reply:
If I remember correctly, the result of the last discussion was that keyspace 0 is just an ordinary keyspace ID, and there should be no problem without special treatment?